zebra/rt_netlink.c

```diff
@@ -2702,7 +2702,7 @@ netlink_put_route_update_msg(struct nl_batch *bth, struct zebra_dplane_ctx *ctx)
 			 */
 			if (RSYSTEM_ROUTE(dplane_ctx_get_type(ctx))
 			    && !RSYSTEM_ROUTE(dplane_ctx_get_old_type(ctx)))
-				netlink_batch_add_msg(
+				return netlink_batch_add_msg(
```

Comment:
Is there a similar issue with the other half of this clause - the v6 half - where we could lose a result? Maybe we could at least report the failure?

Reply:
nope cause we can't return in the second half there.  If we did the delete than add would not work.